controllers/articles.controller.js

```diff
@@ -37,3 +38,20 @@ exports.getArticleComments = (req, res, next) => {
       next(err);
     });
 };
+exports.postArticleComment = (req, res, next) => {
+  const {
+    body,
+    params: { article_id },
+  } = req;
+  Promise.all([
+    articlesModel.checkArticleExists(article_id),
+    usersModel.checkUserExists(body.username),
+    articlesModel.postArticleCommentModel(body, article_id),
+  ])
```

Comment:
You don't really need to check if the user or article exist here - as they are FOREIGN KEYS in the comments table, if they don't exist, the db.query promise will be rejected for you.